nixos/modules/misc/apropos.nix

```diff
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.documentation.apropos;
+
+in
+
+{
+
+  options.documentation.apropos = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to crete an apropos cache for the generated system path.
+        '';
+      };
+  };
+
+  config = mkIf cfg.enable {
+
+    systemd.tmpfiles.rules = [ "d /var/cache/man 0755 root root -" ];
+
+    system.activationScripts.apropos = ''
+      ${pkgs.mandb}/bin/mandb
```

Comment:
Can we do this at build-time instead? Would be much cleaner. This here will run on every boot and rebuild.

Reply:
@matthewbauer are you still interested in this PR? If not I'm ok taking over this task.

Reply:
@rnhmjoj can you please have a look? It would be great to fix this old issue. Thank you.